packages/core/src/Modules/Cashier/Containers/withdrawal.jsx

```diff
@@ -8,12 +8,24 @@ import NoBalance from '../Components/Error/no-balance.jsx';
 import Virtual from '../Components/Error/virtual.jsx';
 import WithdrawalLocked from '../Components/Error/withdrawal-locked.jsx';
 import CashierLocked from '../Components/Error/cashier-locked.jsx';
+import USDTSideNote from '../Components/usdt-side-note.jsx';
 
 class Withdrawal extends React.Component {
     componentDidMount() {
         this.props.setActiveTab(this.props.container);
     }
 
+    componentDidUpdate(prev_props) {
```

Comment:
Since this component is rather small, can we refactor this to use `hooks` via `useEffect` instead of `componentDidMount` and `componentDidUpdate`?

Reply:
Sure.